src/containers/GeneralInfo/NotificationBell/NotificationBell.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import NotificationsActive from "@material-ui/icons/NotificationsActiveOutlined";
+import NotificationsNone from "@material-ui/icons/NotificationsNone";
+
+export interface INotificationBell {
+    hasEnabledNotifications: boolean;
+    onClick?: () => void;
+}
+
+export const NotificationBell: React.FunctionComponent<INotificationBell> = ({
```

Comment:
Either use classNames to avoid this logic that you turned into new component or don't use props but use dispatch and state here. No point in semi-separating logic.